src/types.ts

```diff
@@ -328,3 +328,7 @@ export enum Key {
 export interface CheckboxHandles {
   focus(): void;
 }
+
+export interface BannerHandles {
```

Comment:
Everything in src/types is publicly exported, I'd like to avoid placing extra stuff in here if at all possible (as then changing /moving this stuff becomes a breaking change).

If BannerHandles is only used within the Banner component, can it be defined in there too please?

Reply:
Right. I put it here to match the Checkbox’s but agree. 👍